### ISSUE TITLE
Deprecated config variables from StatAnalysis not found in deprecated conf list

### ISSUE THREAD

**Issue by georgemccabe** [dtcenter/METplus#358]
When a config variable name changes, it must be added to the deprecated_dict list in met_util.py:check_for_deprecated_config function. This is required to allow METplus to report if an old config name is used and to include the changes in the validate_config.py script. I was informed by a user that there are name changes in StatAnalysis wrapper that were not included in this list.

**Comment by georgemccabe** [comment]
List of changes to StatAnalysis and MakePlots provided by Mallory are here:
https://docs.google.com/spreadsheets/d/1dlOtq8BSQwPUl0TRriAd25ix0O5mY__-tS9oIDBVUc0/edit?ts=5e29b022#gid=0

**Comment by georgemccabe** [comment]
I added all of the items from Mallory's list for StatAnalysis and MakePlots to the deprecated item dictionary. Some of these items have a number in them, like MODEL1_NAME or MODEL2_NAME, so I added functionality to look for the string \<n\> in the deprecated item list and find all instances that have a number substituted for \<n\>. 

Related to this work, Logan from EMC discovered that his MET config files were using environment variables set by METplus 2.1 that are no longer set in 3.0. I added functionality to look for *_CONFIG_FILE in the METplus config and check each of those files for deprecated env vars that are set, such as FCST_VAR and OBS_VAR. I updated validate_config.py to prompt the user to replace old environment variables with the new equivalent if appropriate.

Merged into develop.
